public/app/core/services/backend_srv.ts

```diff
@@ -314,17 +314,13 @@ export class BackendSrv implements BackendService {
       return;
     }
 
-    let description = '';
-    let message = err.data.message;
-
-    if (message.length > 80) {
```

Comment:
I feel like there _has_ to be here for a reason for this check to have existed 👀

Reply:
I think its broken so I removed it. If the message is `< 80` then the error that's shown in the UI is just `Validation failed`. You won't see the actual error from the API.

Reply:
I'm pretty sure what's happening here before is to move the error to the "description" of an alert if it's considered "too long" and show "Error" as the title.

In this screenshot you can see an alert with "title and description" and how it differs from "just a title".

<img width="659" alt="image" src="https://user-images.githubusercontent.com/868844/215592402-f1a31eb4-edb6-4bef-ab22-b23d3af0c055.png">

I'm not super comfortable merging these changes - can we pair up to figure out why it doesn't seem to be working for you without the changes proposed in this PR?

Reply:
The issue is that if we keep the following conditional:

```js
if (message.length > 80) {
      description = message;
      message = 'Error';
}
```

Then the error message is never shown to the user because it is shorter than 80 characters:

![Screenshot 2023-01-31 at 10 32 27](https://user-images.githubusercontent.com/85952834/215736226-256c3c4d-0f47-43e2-8248-d95bc5d68ac1.png)

Reply:
That seems like either a regression of the alert component or this has never worked at all 🤔

Reply:
I would guess it has never worked at all, we just didn't know as the API never returned an error message shorter than 80 characters for HTTP 422

Reply:
Oh I see what's going on here, I think there's a better way to solve this for `422` status code errors without impacting the existing logic for the other status codes.